src/dataDisplay/personCoreMilestones/__test__/personCoreMilestones.test.js

```diff
@@ -24,6 +28,10 @@ describe('<PersonCoreMilestones />', () => {
             firstSmallGroupJoinDate: '2001-10-10T12:00:00',
             gender: 'm',
             hasAcceptedChrist: false,
+            hasAttendedActivate1: false,
+            hasAttendedActivate2: false,
+            hasAttendedActivate3: false,
+            hasAttendedActivate4: false,
```

Comment:
These should be `true` if the dates are not null above (Lines 16-19).

Reply:
Fixed: https://github.com/saddlebackdev/react-cm-ui/pull/496/commits/e5f61c48bcb69779107193319f469bc7cb14973b#diff-7866c24dfc448187ccca89fee5c907410233ca734e3ed2fed807712f6db64f90L31-R34